app.js

```diff
@@ -301,9 +301,11 @@ $(document).ready(function () {
 
     APP.translation.init(settings.getLanguage());
 
-    APP.API.init(APP.tokenData.externalAPISettings);
-    // For remote control testing:
-    // APP.API.init({forceEnable: true});
+    APP.API.init(APP.tokenData.jwt ? {
```

Comment:
What is the meaning of the code here ? If there is jwt token then force enable the API and enable the events listed ? Would you mind writing a comment here on why is that necessary ? Without seeing the rest of the commit I guess that it's required for remote control, but that's only, because I know I'm in the context of remote control PR. Other readers will have no idea.